python/addBoat.py

```diff
@@ -14,23 +14,30 @@
 
 
 def command_callback(msg):
+    ships = rospy.wait_for_message('/AIS', AISMsg).ships
+    newShipID = 0
```

Comment:
This is great and definitely works, but you can try this more "pythonic" way

`newShipID = max([ship.ID for ship in ships]) + 1`